doc/changelog/pendings/changes_1624773257463469100.md

```diff
@@ -0,0 +1,9 @@
+### Removed
+- Old gossips api [#5414](https://github.com/azerothcore/azerothcore-wotlk/pull/5414)
+
+### How to upgrade
+- `player->ADD_GOSSIP_ITEM(whatever)` -> `AddGossipItemFor(player, whatever)`
+- `player->ADD_GOSSIP_ITEM_DB(whatever)` -> `AddGossipItemFor(player, whatever)`
+- `player->ADD_GOSSIP_ITEM_EXTENDED(whatever)` -> `AddGossipItemFor(player, whatever)`
+- `player->CLOSE_GOSSIP_MENU()` -> `CloseGossipMenuFor(player)`
+- `player->SEND_GOSSIP_MENU(textid, creature->GetGUID())` -> `SendGossipMenuFor(player, textid, creature->GetGUID())`
```

Comment:
```suggestion
- `player->ADD_GOSSIP_ITEM(whatever)` -> `AddGossipItemFor(player, whatever)`
- `player->ADD_GOSSIP_ITEM_DB(whatever)` -> `AddGossipItemFor(player, whatever)`
- `player->ADD_GOSSIP_ITEM_EXTENDED(whatever)` -> `AddGossipItemFor(player, whatever)`
- `player->CLOSE_GOSSIP_MENU()` -> `CloseGossipMenuFor(player)`
- `player->SEND_GOSSIP_MENU(textid, creature->GetGUID())` -> `SendGossipMenuFor(player, textid, creature->GetGUID())`

You also need  `#include "ScriptedGossip.h"` in your cpp files
```

Maybe it's worth mentioning the file to include, although it could be added to a common header (but not sure)

Reply:
Good idea